src/scancode/cli.py

```diff
@@ -458,11 +465,30 @@ def scancode(ctx,
             full_root=full_root,
             ignore=user_ignore)
 
+        # Find all scans that are both enabled and have a valid function
+        # reference. This deliberately filters out the "info" scan
+        # (which always has a "None" function reference) as there is no
+        # dedicated "infos" key in the results that "plugin_only_findings.has_findings()"
+        # could check.
+        # FIXME: we should not use positional tings tuples for v[0], v[1] that are mysterious values for now
+        active_scans = [k for k, v in scanners.items() if v[0] and v[1]]
+
+        for option, post_scan_handler in plugincode.post_scan.get_post_scan_plugins().items():
+            if kwargs[option.replace('-', '_')]:
+                options['--' + option] = True
+                if not quiet:
+                    echo_stderr('Running post-scan plugin: %(option)s...' % locals(), fg='green')
+                results = post_scan_handler(active_scans, results)
+
+        # FIXME: computing len needs a list and therefore needs loading it all ahead of time
```

Comment:
@pombredanne If I may, wouldn't it be a better approach to rather set a boolean variable which is set to `True` if any `post-scan` plugins are executed.
Then we can check that variable to decide whether or not to recompute `files_count`